Issues in makefile result in the confusing output "@printf: command not found"

What are the steps to reproduce this issue?
-------------------------------------------
1. Use NIC.pl to create a new project
2. Rename Tweak.x to Tweak.xm but do not update makefile
3. $ make package install

What happens?
-------------
Theos outputs the confusing and somewhat useless error message "bash: @printf: command not found"

What were you expecting to happen?
----------------------------------
I think a message of some sort indicating an error in the makefile would be much more helpful as to fixing the issue and not making first time users think that there's a syntax error in their own code

Any logs, error output, etc?
----------------------------
> $ make package install
\> Making all for tweak HomePlus…
bash: @printf: command not found
make[1]: *** [before-HomePlus-all] Error 1
make: *** [HomePlus.all.tweak.variables] Error 2
$ 

Any other comments?
-------------------
Doesn't have to be a verbose error output or anything, but the first time I had this error I figured i'd erroneously included '@printf' somewhere in my tweak.

What versions of software are you using?
----------------------------------------
**Operating System:** macOS Catalina Beta 10.15.1

**Toolchain Version:** Latest at time of submission, I set up THEOS/the entire OS last week

**SDK Version:** 9.2/11.1, issue persists across all installed SDKs and is irrelevant to the one used.

--- a/makefiles/package.mk
+++ b/makefiles/package.mk
@@ -78,12 +78,12 @@ install:: before-install internal-install after-install
 internal-install-check::
 # Throw an error if we aren’t aware of any built package yet.
 	@if [[ -z "$(_THEOS_PACKAGE_LAST_FILENAME)" ]]; then \
-		$(PRINT_FORMAT_ERROR) "$(MAKE) install and show require that you build a package before you try to install it." >&2 \
+		$(PRINT_FORMAT_ERROR) "$(MAKE) install and show require that you build a package before you try to install it." >&2; exit 1; \
 	fi
 
 # Throw an error if the package doesn’t exist.
 	@if [[ ! -f "$(_THEOS_PACKAGE_LAST_FILENAME)" ]]; then \
-		$(PRINT_FORMAT_ERROR) "Could not find “$(_THEOS_PACKAGE_LAST_FILENAME)” to install. Aborting." >&2 \
+		$(PRINT_FORMAT_ERROR) "Could not find “$(_THEOS_PACKAGE_LAST_FILENAME)” to install. Aborting." >&2; exit 1; \
 	fi
 
 export TARGET_INSTALL_REMOTE

--- a/makefiles/package.mk
+++ b/makefiles/package.mk
@@ -78,12 +78,12 @@ install:: before-install internal-install after-install
 internal-install-check::
 # Throw an error if we aren’t aware of any built package yet.
 	@if [[ -z "$(_THEOS_PACKAGE_LAST_FILENAME)" ]]; then \
-		$(ERROR_BEGIN) "$(MAKE) install and show require that you build a package before you try to install it." $(ERROR_END) \
+		$(PRINT_FORMAT_ERROR) "$(MAKE) install and show require that you build a package before you try to install it." >&2 \
 	fi
 
 # Throw an error if the package doesn’t exist.
 	@if [[ ! -f "$(_THEOS_PACKAGE_LAST_FILENAME)" ]]; then \
-		$(ERROR_BEGIN) "Could not find “$(_THEOS_PACKAGE_LAST_FILENAME)” to install. Aborting." $(ERROR_END) \
+		$(PRINT_FORMAT_ERROR) "Could not find “$(_THEOS_PACKAGE_LAST_FILENAME)” to install. Aborting." >&2 \
 	fi
 
 export TARGET_INSTALL_REMOTE

